packages/database/src/migrations/20210618033026-PGStriveAverageMortalityMatrixTable-modifies-data.js

```diff
@@ -0,0 +1,164 @@
+'use strict';
+
+import { generateId, insertObject, arrayToDbString } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const permissionGroupNameToId = async (db, name) => {
+  const record = await db.runSql(`SELECT id FROM permission_group WHERE name = '${name}'`);
+  return record.rows[0] && record.rows[0].id;
+};
+
+const REPORT = {
+  id: generateId(),
+  code: 'PG_Strive_Mosquito_Mortality',
+  config: {
+    fetch: {
+      dataGroups: ['STRVEC_AE-IR', 'STRVEC_AN-IR'],
+      aggregations: [
+        {
+          type: 'RAW',
+          config: {
+            dataSourceEntityType: 'facility',
+          },
+        },
+      ],
+      dataElements: ['STRVEC_AE-IR09', 'STRVEC_AE-IR02', 'STRVEC_AN-IR09', 'STRVEC_AN-IR02'],
+    },
+    output: {
+      type: 'matrix',
+      rowField: 'insecticide',
+      columns: '*',
+      categoryField: 'mosquitoType',
+    },
+    transform: [
+      {
+        '...': ['STRVEC_AE-IR09', 'STRVEC_AN-IR09', 'orgUnitName'],
+        transform: 'select',
+        "'insecticide'":
+          "exists($row['STRVEC_AE-IR02']) ? $row['STRVEC_AE-IR02'] : $row['STRVEC_AN-IR02']",
+        "'mosquitoType'": "exists($row['STRVEC_AE-IR09']) ? 'Aedes IR' : 'Anopheles IR'",
```

Comment:
```suggestion
        '...': ['orgUnitName'],
        transform: 'select',
        "'insecticide'":
          "exists($row['STRVEC_AE-IR02']) ? $row['STRVEC_AE-IR02'] : $row['STRVEC_AN-IR02']",
        "'mosquitoType'": "exists($row['STRVEC_AE-IR09']) ? 'Aedes IR' : 'Anopheles IR'",
        "'mortalityRate'": exists($row['STRVEC_AE-IR09']) ? $row['STRVEC_AE-IR09'] : $row['STRVEC_AN-IR09']
```
Optional, but you can do this if you want to not need to refer to both `STRVEC_AE-IR09` and `STRVEC_AE-IR09` throughout the rest of the transform logic if ya want? Would probably make it a bit more readable too 👍

Reply:
That's brillant 👍 . I definitely want it, thanks.